### PR TITLE
Add bounding box argument to level 2 chunk graph endpoint

### PR DESCRIPTION
**Pull request by bdpedigo**
Client-side companion PR: https://github.com/seung-lab/CAVEclient/pull/154

### CODE REVIEW

File: pychunkedgraph/app/segmentation/common.py [hunk 1]
```diff
@@ -1042,9 +1042,15 @@ def handle_get_layer2_graph(table_id, node_id):
     current_app.table_id = table_id
     user_id = str(g.auth_user.get("id", current_app.user_id))
 
+    if "bounds" in request.args:
```
Comment (fcollman) [inline pychunkedgraph/app/segmentation/common.py:1045]: don't we have a common function for parsing this argument? if we don't we should

File: pychunkedgraph/graph/analysis/pathing.py [hunk 2]
```diff
@@ -70,14 +73,40 @@ def get_children_at_layer(
...
     """get an edge list of lvl2 ids for a particular node
 
     :param cg: ChunkedGraph object
     :param node_id: np.uint64 that you want the edge list for
+    :param bbox: Optional[Sequence[Sequence[int]]] a bounding box to limit the search
     """
 
-    lvl2_ids = get_children_at_layer(cg, node_id, 2)
+    if bbox is None:
+        # maybe temporary, this was the old implementation
```
Comment (fcollman) [inline pychunkedgraph/graph/analysis/pathing.py:89]: if we expect this to work perhaps we should just put in a test and make this code simplier.. on the other hand this is guarunteed to not be a breaking change so it's not a bad way to go.

Comment (bdpedigo) [reply pychunkedgraph/graph/analysis/pathing.py:89]: elected to keep it this way with the comment for now, but in theory the other function should handle both cases

**Review by akhileshh — APPROVED** [review]
lgtm

**Pull request merged**